echo-web/config/echo.yml

```diff
@@ -2,16 +2,24 @@ server:
   port: 8089
 
 spinnaker:
+  inMemory:
```

Comment:
Do you know where this is used?  I suppose it doesn't hurt anything to have this here if it's unused, but I couldn't find this ever being read.

Reply:
removing this, I also don't see it being used